src/pdsfile/holdings_maintenance/_archives_common.py

```
@@ -0,0 +1,241 @@
+
+        for (abspath, dirpath, nbytes, modtime) in dir_tuples:
+            if abspath not in tardict:
+                logger.error('Missing from tar file', abspath)
+                valid = False
+
+            elif (dirpath, nbytes, modtime) != tardict[abspath]:
+
+                if nbytes != tardict[abspath][1]:
+                    logger.error('Byte count mismatch: ' +
+                                 str(nbytes) + ' (filesystem) vs. ' +
+                                 str(tardict[abspath][1]) + ' (tarfile)', abspath)
```

Ruff (0.16.1)

[warning] Logging statement uses `+`

(G003)